packages/theming/win32-theme/src/__tests__/win32-theme.test.ts

```diff
@@ -35,7 +35,6 @@ it('createFontAliasTokens test', () => {
 });
 
 it('createOfficeTheme test', () => {
-  setCurrentHostThemeSetting('White');
```

Comment:
I removed this to make sure that the old behavior still works without causing any issues.

The main scenario where this is causing issues is jest tests. The native module isn't available in jest tests so we run our fallback, which means that a host theme isn't set when the win32 theme tries to use it. Since it's valid that clients downstream would use jest/non-host available environments when running our code, it's better to remove the error and keep a test which makes sure that the scenario doesn't cause any trouble.

Reply:
I see, thanks for the explanation!